recipe/meta.yaml

```diff
@@ -101,11 +101,11 @@ outputs:
         - wheel
         - zlib 1.2.11
         # coin or deps
-        - coin-or-utils 2.11.4
-        - coin-or-osi 0.108.6
-        - coin-or-clp 1.17.4
-        - coin-or-cgl 0.60.3
-        - coin-or-cbc 2.10.5
+        - coin-or-utils
+        - coin-or-osi
+        - coin-or-clp <=1.17.4
```

Comment:
I think this works (because we don't have any other versions available), but it's not "correct". I think it should really be `=1.17.4` or have at least a lower bound of something like `>=1.17,<=1.17.4`